docs/sdk/api/sdkMiddlewareAuth.md

```diff
@@ -150,3 +150,30 @@ const client = createClient({
   ],
 })
 ```
+
+## `createAuthMiddlewareForExistingToken(options)`
+
+Creates a [middleware](/sdk/Glossary.md#middleware) that attaches a known access token a request header.
+
+#### Named arguments (options)
+
+`options` can be a string or an object. If `options` is a _(String)_, it is treated as the access token with other possible options set to their defaults. If `options` is an _(Object)_, it can have the following properties:
+
+1. `token` _(String)_: the access token to be attached to the request
+2. `tokenType` _(String)_: the type of the access token (Default: `Bearer`)
```

Comment:
Does the API accept anything else than a Bearer?

Reply:
No. I left it configurable in the event that there is a future supported token type. `Bearer` is still the default

Reply:
It would be good if we keep those middlewares as agnostic as possible in regards to our API. After all, the `sdk-client` can be used by any API.